connect/transforms/src/main/java/org/apache/kafka/connect/transforms/SetSchemaMetadata.java

```diff
@@ -154,4 +161,14 @@ protected static Object updateSchemaIn(Object keyOrValue, Schema updatedSchema)
         }
         return keyOrValue;
     }
+
+    private static String buildSchemaName(String namespace, String name) {
+        if (Utils.isBlank(namespace)) {
+            return name;
+        } else if (!Utils.isBlank(name)) {
+            return namespace + "." + name;
```

Comment:
What happens if the `namespace` ends with a `.`? We should probably handle that case more transparently. Unit tests should verify that `schema.namespace=a.b.c.` and `schema.namespace=a.b.c` are treated the same.